ajax/libs/asynquence-contrib/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "asynquence-contrib",
+  "version": "0.27.0",
+  "description": "additional plugins for asynquence",
+  "filename": "contrib.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/getify/asynquence.git"
+  },
+  "keywords": [
+    "async",
+    "flow-control",
+    "sequences",
+    "promise",
+    "iterator",
+    "generator"
+  ],
+  "author": "Kyle Simpson <getify@gmail.com>",
+  "license": "MIT",
+  "npmName": "asynquence-contrib",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "contrib+(-common*|-es6*|.src|).js"
```

Comment:
@pvnr0082t  but that will catch all file match `contrib!(-wrapper)*`, the author clearly said only need that three file(`contrib.js`, `contrib-common.js`, and `contrib-es6.src.js`), so I think we should be more careful in this case to avoid adding files we don't need.